examples/two-columns-layout/style.css

```
@@ -316,12 +316,133 @@ body {
   height: var(--aa-spacing);
 }
 
+/* Quick access */
+.aa-PanelSection--quickAccess .aa-List {
+  display: flex;
+  column-gap: var(--aa-spacing-half);
+  align-items: stretch;
+}
+
+.aa-PanelSection--quickAccess .aa-Item {
+  width: 100%;
+}
+
+.aa-QuickAccessItem {
+  height: 100%;
+}
+
+.aa-QuickAccessItem.aa-ItemLink {
```

I don't think this is necessary.

```suggestion
.aa-QuickAccessItem {
```